recipes/ploomber/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "ploomber" %}
+{% set version = "0.11.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0dab5e33e76b94c33c2c21145c60e965f8d87b64034d08ed553acbeab16eb3b4
+
+build:
+  number: 0
```

This looks like the only reason this isn't noarch: python is because of the conditional dependency on `importlib_resources`. Better to just unconditionally depend on that and:
```suggestion
  number: 0
  noarch: python
```